### PR TITLE
Update @ckeditor/ckeditor5-link to v28

### PR DESCRIPTION
**Pull request by fedemp**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/ckeditor/ckeditor5/tree/e67034923ab25c7c12782e57b2a10434391631a6/packages/ckeditor5-link
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/ckeditor__ckeditor5-link/tsconfig.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
     "compilerOptions": {
         "module": "commonjs",
-        "target": "es6",
```
Comment (peterblazejewicz) [inline types/ckeditor__ckeditor5-link/tsconfig.json:4]: 👍🏻

**Comment by peterblazejewicz** [comment]
@fedemp do you think this is maintainable to keep versioning here in the DT source? (v27 is already published t NPM via @types/..). Or do you expect to make any changes to v27 in future?
https://github.com/ckeditor/ckeditor5/tags

**Comment by fedemp** [comment]
I'm sorry but I don't understand the question. Do you mean that we should ignore the `v27` directory and only publish types for `v28`?

> Or do you expect to make any changes to v27 in future?

Users will surely use latest version, not a previous one. The only reason to update v27 is if I find a bug while typing v28 that was carried over from v27.

**Comment by peterblazejewicz** [comment]
I'm only concerned about the maintenance cost of having more than one version. That's up to you though, just wanted to know your opinion, thx!

**Review by peterblazejewicz — APPROVED** [review]
LGTM!

**Comment by fedemp** [comment]
Ready to merge

**Pull request merged**